public/ng-views/maps.html

```diff
@@ -550,78 +621,109 @@ <h3 class="noTopMargin noBottomMargin">
           Normal
         </label>
         <label  class="btn btn-info" btn-radio="1"
-                ng-model="scenario[tabnav].station"
+                ng-model="scenario[tabnav].station" 
                 ng-click="scenario[tabnav].station=1;
                           updateRouteScorecard(scenario[tabnav].routeId, tabnav)">
-          <i class="fa fa-star"></i>
-          <i class="fa fa-star"></i>
-          <i class="fa fa-star-o"></i>
+          <img class="stationIcons" src="public/imgs/platform_black_yellow.svg">
           Platform
         </label>
         <label  class="btn btn-info" btn-radio="2"
-                ng-model="scenario[tabnav].station"
+                ng-model="scenario[tabnav].station" 
                 ng-click="scenario[tabnav].station=2;
                           updateRouteScorecard(scenario[tabnav].routeId, tabnav)">
-          <i class="fa fa-star"></i>
-          <i class="fa fa-star"></i>
-          <i class="fa fa-star"></i>
+          <img class="stationIcons" src="public/imgs/brt_black_yellow.svg">
           BRT
         </label>
         <label class="btn btn-success" ng-click="saveAlt=true; createVar=false">
-          <i class="fa fa-cloud-upload"></i> Save Variation
+          <i class="fa fa-cloud-upload"></i> Save
         </label>
       </div>
     </div>
   </div>
 
 </div>
+<!--END of ROUTE EDITOR MODULE--> 
 
+<div class="controlModules">
 
 
-<div class="basecol">
+<!--POI MARKER ACTIVATION --> 
+
+<div class="basecol_POI col-md-3 "> 
+<!-- <h5 style="margin-bottom: 10px"> PLACES </h5> -->
+  <div class="fa-stack" ng-click="targetPOIUsers(); poiUserDots = !poiUserDots" style="margin-bottom: 8px">
+    <i class="fa fa-circle fa-stack-2x"></i>
+    <i class="fa fa-users fa-stack-1x fa-inverse destinationIcon iconContainer"></i>
+  </div>
+  <br>
+  <span ng-show="poiUserDots">
+    <span class="fa-stack userDot" 
+          ng-repeat="user in poiUsers track by $index" 
+          ng-click="targetPOIUsers(user.name)" 
+          ng-class="user.name == currentPOIUser ? '' : 'dot-unselected'"
+          style="color:#{{ user.color }}">
+      <i class="fa fa-circle fa-stack-2x"></i>
+      <i class="fa fa-stack-1x fa-inverse">
+        <small>{{ user.name }}</small>
+      </i>
+    </span>
+  <span>
+
+</div>
+<!--End of POI MARKER ACTIVATION --> 
+
+
+<!--DESTINATION & TIME FILTER --> 
+
+<div class="basecol_filter col-md-3 ">
 
   <div class="progress" ng-show="loadProgress.vis">
-    <div class="progress-bar progress-bar-info progress-bar-striped active"
+    <div class="progress-bar progress-bar-info progress-bar-striped active" 
           role="progressbar"  id="isosProgressBar"
           style="width:{{loadProgress.val}}%">
       {{loadProgress.val}}%
     </div>
   </div>
+  <h5 style="margin-bottom: -7px">
+    TRAVEL TIME: 
+    <span ng-click="toggleShowVectorIsos()">
+            {{ vectorIsos.val*5 | minuteConverter }}
+            <span class="badge">
```

Comment:
Why did you leave the badge class in or did you add it? Since this is a vertical design that is dangerous because it is an inline element so unless the whole thing is nested with overrides for overflow, then this is improper:

![image](https://cloud.githubusercontent.com/assets/6053396/8777669/700ab354-2ec7-11e5-8bb2-83660947e5cd.png)